ietf/templates/meeting/upcoming.ics

```diff
@@ -4,7 +4,7 @@ METHOD:PUBLISH
 PRODID:-//IETF//datatracker.ietf.org ical upcoming//EN
 {{vtimezones}}{% for item in assignments %}BEGIN:VEVENT
 UID:ietf-{{item.session.meeting.number}}-{{item.timeslot.pk}}
-SUMMARY:{% if item.session.name %}{{item.session.name|ics_esc}}{% else %}{{item.session.group.acronym|lower}} - {{item.session.group.name}}{%endif%}
+SUMMARY:{% if item.session.name %}{{item.session.group.acronym|lower}} - {{item.session.name|ics_esc}}){% else %}{{item.session.group.acronym|lower}} - {{item.session.group.name}}{%endif%}
```

Comment:
There seems to be a stray `)` in this line. On my test, where I gave a session the name "Named Session," I get:
```
[...]
BEGIN:VEVENT
UID:ietf-interim-2022-cellar-09-14114
SUMMARY:cellar - Named Session)
STATUS:CONFIRMED
[...]
```

Reply:
good catch